modules/ROOT/pages/_partials/sync/server-files.adoc

```diff
@@ -0,0 +1,54 @@
+= File Upload Server
+
+{sync-server} provides support for uploading binary data along with the GraphQL queries.
+Implementation is relying on upstream `Apollo NodeJS` server capabilities
```

Comment:
I'm not 100% sure but are you saying it relies on capabilities provided by the apollo-server package? If so my suggestion is this:

```suggestion
Implementation relies on upstream `Apollo Server` capabilities.
```